src/ip/Address.h

```diff
@@ -44,10 +44,10 @@ class Address
     /** @name Constructors */
     /*@{*/
     Address() { setEmpty(); }
+    Address(const struct sockaddr_storage &);
     Address(const struct in_addr &);
     Address(const struct sockaddr_in &);
     Address(const struct in6_addr &);
-    Address(const struct sockaddr_in6 &);
```

Comment:
I support adding a sockaddr_storage constructor, but why remove this constructor? What is wrong with it (other than the fact that it allows implicit conversions -- a problem that should be fixed by adding "explicit" rather than by removing)?

PR description says "Migration to sockaddr_storage API", but that phrase does not answer my question (and raises more red flags). If that phrase was meant to explain why sockaddr_in6 conversion constructor is removed, please rephrase. Otherwise, please remove that PR description phrase completely, so that we do not have to argue about (and adjust it to clarify) its meaning.

Reply:
This constructor (and matching assignment operator) are no longer used after this PR updates the callers to `sockaddr_storage`.